backend/src/LibExecution/RuntimeTypesToDarkTypes.fs

```diff
@@ -337,6 +339,8 @@ module TypeReference =
     | DEnum(_, _, [], "TUnit", []) -> TUnit
     | DEnum(_, _, [], "TBool", []) -> TBool
     | DEnum(_, _, [], "TInt", []) -> TInt
+    | DEnum(_, _, [], "TInt8", []) -> TInt8
```

Comment:
Can you verify that we have tests to roundtrip these? If not, can you add them as a followup PR?

Reply:
Yes, we do